paymentsheet/src/test/java/com/stripe/android/paymentsheet/viewmodels/PaymentOptionsStateMapperTest.kt

```diff
@@ -38,9 +38,6 @@ class PaymentOptionsStateMapperTest {
         mapper().test {
             assertThat(awaitItem()).isNull()
 
-            paymentMethodsFlow.value = PaymentMethodFixtures.createCards(2)
-            assertThat(awaitItem()).isNull()
-
```

Comment:
Seems like we were pretty explicit about this behavior. Do we have any context as to why?

Reply:
This is weird. I have no idea.

Reply:
Looks like this also changed recently, and you need a rebase.

Reply:
Looks like I had an old branch with the same name.